python_modules/libraries/dagster-slack/dagster_slack/resources.py

```diff
@@ -1,29 +1,63 @@
-from dagster import Field, StringSource, resource
+from dagster import ConfigurableResource, resource
+from pydantic import Field
 from slack_sdk.web.client import WebClient
 
 
+class SlackResource(ConfigurableResource):
+    """This resource is for connecting to Slack.
+
+    The resource interacts with Slack via `slack_sdk.WebClient`.
```

Comment:
Also I think this language is slightly off. I returns a `slack_sdk.WebClient` from `get_client`. This sentence makes it sound like it calls the web client's methods on your behalf, which it doesn't.

Reply:
im going to get rid of this line and move it to down to docstring of `get_client`